### PR TITLE
SPDX Defects Team meeting - 9th March 2022

### PR DESCRIPTION
**Pull request by rnjudge**
Signed-off-by: Rose Judge <rjudge@vmware.com>

### CODE REVIEW

File: defects/2022-03-09.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+Thomas: CPEs are not usable, highly computational
+* Karsten's main concern is dynamic information being included in static document.
+* Thomas: as a supplier, we don't know the CPEs because the publisher of the package does not provide it. There's a mismatch -- this is why package URLs can be useful (they are computational). 
+* Dick: add third option that Karsten is describing into the list.
+* Thomas: To recap Karsten proposal add a 3rd option that makes external package identifier CPE or purl mandatory when sharing security information.
+* Thomas/Rose: If using option 3, what if you have a vulnerability but doesn't have a CVE number?
+* Dick: component and vulnerability that affect a particular SBOM can be in the URL
+* Karstens main concern is wanting to separate static and dynamic.
+* Dick: It about giving vendor flexiablity to communicate vulnerabilities from SPDX whether CycloneDX, CSAF, etc.
+* Thomas TODO: Add a 3rd option to SPDX 2.3 documentation
```
Comment (rjb4standards) [inline defects/2022-03-09.md:66]: There was a suggestion to set a goal to reach consensus on one of the 3 options at our next meeting in order to meet the NTIA minimum implementation timeline, currently scheduled for August 2022.

**Review by rjb4standards — CHANGES_REQUESTED** [review]

File: defects/2022-03-09.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+* If we require licensing infromation can lead to [size] inflated SBOMs with lots of NONEs and NOASSERTIONs.
+* Dick agrees a lot of the licensing material is extraneous but he can go either way right now.
+* Jeff: if we are adapting 2.3 to disclose security information about components then we should be able to omit licensing information (Dick agrees).
+* Are we getting complaints/requests for this? 
+* We are waiting for someone to open a PR in the repo for this? Yes. Fields that are related to licensing and have a NOASSERTION option that are currently mandatory should be moved to optional. Rose volunteers to help open this PR.
+  * ConcludedLicense, All License information from package (licenses from file, etc), DeclaredLicense, CopyrightText for packages, files and snippets.
+  * https://spdx.dev/wp-content/uploads/sites/41/2020/04/spdx_onepager.pdf
+* Dick: One concern related to NTIA support for US EA. If we make this change, is it going to create any delay in getting 2.3 out the door before the August deadline? Validation checks will likely need to be re-coded to do this.
```
Comment (jeff-schutt) [inline defects/2022-03-09.md:34]:
```suggestion
* Dick: One concern related to NTIA support for US EO. If we make this change, is it going to create any delay in getting 2.3 out the door before the August deadline? Validation checks will likely need to be re-coded to do this.
```

File: defects/2022-03-09.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+  * Validation tooling will need to be adjusted to not flag if it's missing.
+  * Q: Anyone from tooling that could tell us how big of a change this would be? A: Gary ONeall 
+* Kate shares comparison of NTIA comparison to SPDX fields: (https://docs.google.com/document/d/1fkw_RwdTPH2XvrlLQFanYr_A0JFF6y9KB5HncrcZFYM/edit)
+    * 7.2 Unique identifier NTIA minimum columnn: part 1 you have to have a namespace part 2 is you have to have something relative to that namespace
+    * Kate: NTIA wording for component: unit of software defined by the supplier (package or snippet).
+    * Jeff: While SPDX lists packages as a component, the SPDX document itself recognizes a component which also has its own identifier and document namespace?
```
Comment (jeff-schutt) [inline defects/2022-03-09.md:40]:
```suggestion
    * Jeff: SPDX lists packages as components and each package has a unique identifier in the SPDX document, which meets the NTIA 'other unique identifiers' requirement. Kate is highlighting that the software that SPDX document references is recognized as a component, and so the SPDX document has its own unique identifier by means of the document namespace, also meeting the NTIA requirement.
```

**Review by jeff-schutt — CHANGES_REQUESTED** [review]

File: defects/2022-03-09.md (hunk 2 above)
Comment (rnjudge) [reply defects/2022-03-09.md:34]: Good catch, Jeff! Fixed.

File: defects/2022-03-09.md (hunk 3 above)
Comment (rnjudge) [reply defects/2022-03-09.md:40]: Thanks again, Jeff. I updated my PR to reflect your suggestion.

**Comment by rjb4standards** [comment]
good catch, Jeff. thanx

**Pull request merged**